.github/workflows/osx.yml

```diff
@@ -33,14 +33,15 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: recursive
-      
+
       - uses: actions/setup-python@v5
         with:
-          python-version: '3.12' 
-      
+          python-version: "3.12"
+
       - name: Install dependencies
         run: |
-          HOMEBREW_NO_AUTO_UPDATE=1 brew install autoconf automake protobuf cmake ccache libtool sqlite3 libspatialite luajit curl wget czmq lz4 spatialite-tools unzip boost gdal
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install python autoconf automake protobuf cmake ccache libtool sqlite3 libspatialite luajit curl wget czmq lz4 spatialite-tools unzip boost gdal
+          export PATH="$(brew --prefix python)/libexec/bin:$PATH"
```

Comment:
Only two changes except for whitespace and double quotes formatting:
1) install python via homebrew
2) update $PATH so homebrew python is